sql/sql_plugin.cc

```diff
@@ -128,13 +128,13 @@ static int plugin_type_initialization_order[MYSQL_MAX_PLUGIN_TYPE_NUM]=
 {
   MYSQL_DAEMON_PLUGIN,
   MariaDB_ENCRYPTION_PLUGIN,
+  MYSQL_REPLICATION_PLUGIN,
   MYSQL_STORAGE_ENGINE_PLUGIN,
   MYSQL_INFORMATION_SCHEMA_PLUGIN,
   MYSQL_FTPARSER_PLUGIN,
   MYSQL_AUTHENTICATION_PLUGIN,
   MariaDB_PASSWORD_VALIDATION_PLUGIN,
   MYSQL_AUDIT_PLUGIN,
-  MYSQL_REPLICATION_PLUGIN,
```

Comment:
This change looks suspicious. Native MYSQL_REPLICATION_PLUGIN may need storage engine plugins to be initialized first.

Reply:
WSREP plugin is now type of WSREP_PLUGIN. Initial ordering is still preserved.